roq-management/src/main/java/org/roqmessaging/management/GlobalConfigLeaderListener.java

```diff
@@ -0,0 +1,28 @@
+package org.roqmessaging.management;
+
+import org.apache.curator.framework.recipes.leader.LeaderLatchListener;
+
+/**
+ * This class implements the interface LeaderLatchListener
+ * of Curator. It allows to register a listener which
+ * handle master change when active master is running.
+ * That allows to handle the ZK connection lost.
+ * 
+ * @author BVM
+ *
+ */
+public class GlobalConfigLeaderListener implements LeaderLatchListener {
+	@Override
+	public void notLeader() {
+		// Notifies that the GCM has lost the lead
+		GlobalConfigurationManager.hasLead = false;
```

Comment:
Why is ```hasLead``` a static variable of the GCM? I think that is a mistake, but maybe there's an explanation.